jsons/Policies.json

```diff
@@ -195,7 +195,7 @@
 			},
 			{
 				"name": "Religious Tolerance",
-				"uniques": ["[+1 Culture] [in all cities]","+[25]% [Science] from every [Grand Temple]","[+2 Science] from every [Temple]"],
+				"uniques": ["[+1 Culture] [in all cities]","+[25]% Science from every [Grand Temple]","[+2 Science] from every [Temple]"],
```

Comment:
Same here

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c